README.rst

```diff
@@ -8,9 +8,8 @@ A python implementation of elastic-net regularized generalized linear models
 
 `[Documentation (stable version)]`_ `[Documentation (development version)]`_
 
-.. raw:: html
-
-    <img src="pyglmnet-logo.png" height="400px">
+.. image:: https://raw.githubusercontent.com/glm-tools/pyglmnet/master/pyglmnet-logo.png
+   :height: 400px
```

Comment:
@pavanramkumar what do we do about this?

The height parameter is unfortunately not respected by github which is why we had to use the raw::html directive

Reply:
let's maybe use this so we don't have to use the `height` parameter:

![pyglmnet](https://user-images.githubusercontent.com/15852194/67919367-70482600-fb76-11e9-9b86-891969bd2bee.jpg)

The url would be: https://user-images.githubusercontent.com/15852194/67919367-70482600-fb76-11e9-9b86-891969bd2bee.jpg

Reply:
@jasmainak seems like a reasonable solution to use a smaller image. is that was you proposed?